src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,30 @@
+.catalog {
+  --columns: 1;
+
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(var(--columns), $card-width);
+  gap: $catalog-gap;
+
+  @media (min-width: 488px) and (max-width: 767px) {
+    --columns: 2;
+  }
+
+  @media (min-width: 768px) and (max-width: 1023px) {
+    --columns: 3;
```

Comment:
i think you don`t need max-width here, because your next min-width is working like max.